tasks/main.yml

```
@@ -58,6 +58,20 @@
   loop_control:
     label: "{{ item.section }}:{{ item.option }}"
 
+- name: Place TLS certificates
+  ansible.builtin.copy:
+    src: "{{ item.src }}"
+    dest: "{{ item.dst }}"
+    owner: bareos
+    group: bareos
+    mode: "0640"
+    backup: "{{ bareos_dir_backup_configurations }}"
+  loop:
+    - { src: "ca.pem", dst: "{{ bareos_webui_tls_ca_cert }}" }
+    - { src: "chain.pem", dst: "{{ bareos_webui_tls_cert_chain }}" }
+  when:
+    - bareos_webui_tls_enable == "yes"
```

don't compare boolean values like strings. you can just go `when: yourBoolean` as it's either true or false.

```suggestion
    - bareos_webui_tls_enable
```